baselines/disable-packing-test/src/v1beta1/echo_client.ts.baseline

```diff
@@ -160,7 +160,7 @@ export class EchoClient {
       userPathTemplate: new this._gaxModule.PathTemplate(
         'users/{user_id}'
       ),
-      userIdBlurbIdPathTemplate: new this._gaxModule.PathTemplate(
+      userIdBlurbIdBlurbPathTemplate: new this._gaxModule.PathTemplate(
```

Comment:
I understand why this one has changed, but can we make the naming logic so that it would get something like `userIdProfileBlurbId` instead? Will make more sense I guess.

Reply:
The logic would be, for pairs `names/{name}`, take `name`; for parts without subsequent `{name}`, take the part name as is. So it would take `userId` for `user/{user_id}`, then `profile`, then `blurbId` for `blurbs/{blurb_id}`.

Reply:
yes, that's a side effect, I tried some ways to fix this to make it more readable but haven't get any perfect solution. I will dig deeper and try more. Thanks

Reply:
sounds reasonable to me, so for pattern `user/{user_id}/profile/blurbs/{blurb_id}`, name will be changed to `userId_profile_blurbId`.